html_css/js/main.js

```diff
@@ -1,8 +1,15 @@
-$(document).ready(function(){
-    $('.nav-btn').click(function(){
-        $('.responsive-nav').toggleClass('active');
-    });
-});
+const navbtn = document.querySelector(".nav-btn");
+const sidemenu = document.querySelector(".sidemenu");
+
+navbtn.addEventListener("click", () => {
+    navbtn.classList.toggle("active");
+    sidemenu.classList.toggle("active");
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/aerial978/blog/pullRequest?prid=9584603)
```suggestion
  sidemenu.classList.toggle("active");
```